recipes/torchaudio/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set build_variant = environ.get('CONDA_BUILD_VARIANT', 'cpu') %}
+{% set name = "audio" %}{% set version = "0.10.0" %}
+{% set torch_proc_type = "cuda" if cuda_compiler_version != "None" else "cpu" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pytorch/audio/archive/v0.10.0.tar.gz
+  sha256: 9d0413434eefd2a483bf4006440c02940d15e597ab75703aa3ae2c6d5297e7ad
+
+build:
+  number: 0
+  skip: true  # [win]
+
+outputs:
```

Comment:
Why are you bothering with outputs? Typically they help fix mistakes or help split a build up into many. I don't think you need them here.

Reply:
> Why are you bothering with outputs? Typically they help fix mistakes or help split a build up into many. I don't think you need them here.

I am not really sure 😶‍🌫️ 

I am testing random ideas to see what sticks. For some reason, when I didn't have the outputs it kept failing. I will try without the outputs soon. I am also really bad at squashing commits, so this is a total mess (I am sorry, but thank you so much for helping!)

Reply:
Oh, now I remember: How do I call the scripts outside the requirements? Like torchvision, there are two scripts that should be called I assume. Or am I reading this wrong?

Reply:
Never mind, I think I can do it like in line 35 of the example

Reply:
I think you just call them `build.sh` and `bld.bat`

See a simple recipe:
https://github.com/conda-forge/check-feedstock/blob/master/recipe/meta.yaml#L19